websockify/websocketproxy.py

```diff
@@ -90,6 +90,20 @@ def validate_connection(self, no_tokens=False):
                 self.send_auth_error(ex)
                 raise
 
+    def do_GET(self):
```

Comment:
Why the move? I made more sense to have it in `WebsockifyServer` to keep symmetry with the existing call to `validate_connection()`.

Reply:
The problem is that I was calling validate_connection(no_tokens=True) in WebSockifyRequestHandler, but the behavior for no_tokens=True was implemented in ProxyRequestHandler, a subclass of WebSockifyRequestHandler. It seemed wrong to make WebSockifyRequestHandler expect any such thing of a subclass, so I just moved everything to ProxyRequestHandler, with the bonus that now the actual functionality for --web-auth is all there in one place.

Reply:
That problem will be avoided if we don't need the extra parameter.

Reply:
Done.